sync-team/src/github/mod.rs

```diff
@@ -865,7 +878,7 @@ struct CreateRepoDiff {
     settings: RepoSettings,
     permissions: Vec<RepoPermissionAssignmentDiff>,
     branch_protections: Vec<(String, api::BranchProtection)>,
-    environments: Vec<String>,
+    environments: Vec<(String, Vec<String>)>,
```

Comment:
I would substitute `Vec<String>` with a struct

Reply:
I would like to change either whole  `Vec<(String, Vec<String>)>` as a struct, not just `Vec<String>`. IMO, it will make it unnecessarily complex. 
```rust
struct Branches {
    names: Vec<String>
}
```


I would like to keep it as it is. WDYT?